app/assess/templates/contract_downloads.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
+{%- from 'govuk_frontend_jinja/components/textarea/macro.html' import govukTextarea -%}
+{% from "macros/banner_summary.html" import banner_summary %}
+{% from "macros/logout_partial.html" import logout_partial %}
+
+{% block content %}
+<div class="govuk-phase-banner flex-parent-element">
+    <p class="govuk-!-text-align-left flex-parent-element flexed-element-margins-collapse">
+        <a href="{{url_for('assess_bp.application', application_id=application_id)}}" class="govuk-back-link"
+            id="back-to-assessment-overview-link">
+            Back to <b>assessment overview</b></a>
+    </p>
+    {{ logout_partial(sso_logout_url) }}
+</div>
+{{ banner_summary(
+fund_name,
+state.short_id,
+state.project_name,
+state.funding_amount_requested,
+state.display_status,
```

Comment:
@gio-karunakaran it is being used here. So we will need to pass `display_status` through to the template from the return value of `determine_display_status`.

Reply:
ahh good spot! wasn't being passed through to the render template function so that it was dead code 👍🏾 have updated this now!